contributors/devel/on-call-federation-build-cop.md

```diff
@@ -0,0 +1,159 @@
+# Federation Buildcop Guide and Playbook
+
+Federation runs two classes of tests: CI and Presubmits.
+
+## CI
+
+* These tests run on the HEADs of master and release branches.
+* As a result, they run on code that's already merged.
+* As the name suggests, they run continuously. Currently, they are
+  configured to run at least once every 30 minutes.
+* Federation CI tests still run on Jenkins and this mode of testing is
```

Comment:
By "this mode" I assume you mean Jenkins? Would be better to be explicit and say "Jenkins is deprecated"